andela/AndelaEatsUI
# #166573002 Remove redundant code and Fix Side Nav

#### What does this PR do?
- Removes the redundant file `../components/Admin/Tabs/MenusAndMeals.jsx`.
- Adds the `Meal Sessions` tab to the Side Navigation.

#### Description of Task to be completed?
- The Side Nav is missing the Meal Sessions tab, and there is redundant code in the Admin directory. This needs to be refactored

#### How should this be manually tested?
- Clone the repo and checkout the `remove-redundant-code-166573002` branch.
- Run the application, as an admin, and click the `manage Menus` tab.
- You should see the `Meal Sessions` tab.
#### What are the relevant pivotal tracker stories?
- [ #166573002](https://www.pivotaltracker.com/story/show/166573002)

#### Background Context
- None

#### Screenshots (If Applicable)
<img width="359" alt="Screen Shot 2019-06-10 at 12 18 14" src="https://user-images.githubusercontent.com/30738053/59185752-e32a1380-8b79-11e9-84aa-4eb218eae0c2.png">
